### PR TITLE
Move HandleDeath to before dragonfly handles the death

### PR DESCRIPTION
**Pull request by Prim69**
This allows users to get the contents of the player's inventory when they died, change the drops when a player dies, etc.

### CODE REVIEW

File: server/player/player.go [hunk 1]
```diff
@@ -806,6 +806,8 @@ func (p *Player) kill(src damage.Source) {
 		viewer.ViewEntityAction(p, entity.DeathAction{})
 	}
 
+	p.handler().HandleDeath(src)
```
Comment (T14Raptor) [inline server/player/player.go:809]: I think this should get called after the health is set to zero for the player.

Comment (Prim69) [reply server/player/player.go]: Done

**Comment by T14Raptor** [comment]
Run go fmt over this then I can merge

**Comment by Prim69** [comment]
All good now

**Comment by T14Raptor** [comment]
Thanks!

**Pull request merged**